### PR TITLE
Refactor image steps

### PR DESCRIPTION
**Pull request by EDbarvinsky**

### CODE REVIEW

File: vividus-tests/src/main/resources/story/integration/ImageStepsTests.story [hunk 1]
```diff
@@ -0,0 +1,74 @@
+Meta:
+
```
Comment (valfirst) [inline vividus-tests/src/main/resources/story/integration/ImageStepsTests.story]: `@group vividus-plugin-web-app`

File: vividus-plugin-web-app/src/main/resources/steps/defaults/image.steps [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+When I hover a mouse over an element with the xpath './/img[@alt='<tooltipImage>' or @title='<tooltipImage>']'
+
+Composite: When I click on an image with the src '$src'
+When I click on an element by the xpath './/img[@src='<src>']'
+
+Composite: When I click on an image with the name '$imageName'
+When I click on an element by the xpath './/img[@*='<imageName>' or text()='<imageName>']'
+
+Composite: Then an image with the src '$src' exists
+Then number of elements found by `By.xpath(.//img[@src='<src>'])` is = `1`
```
Comment (valfirst) [inline vividus-plugin-web-app/src/main/resources/steps/defaults/image.steps]: `is =` -> `is equal to` here and below, to make steps more human-friendly

File: vividus-plugin-web-app/src/main/resources/steps/defaults/image.steps [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+Composite: Then a [$state] image with the src '$src' exists
+Then number of elements found by `By.xpath(.//img[@src='<src>'])->filter.state(<state>)` is = `1`
```
Comment (ikalinin1) [inline vividus-plugin-web-app/src/main/resources/steps/defaults/image.steps]: By.imageSrc(<src>) ?

Comment (ikalinin1) [reply vividus-plugin-web-app/src/main/resources/steps/defaults/image.steps]: Here and everywhere

File: vividus-plugin-web-app/src/main/resources/steps/defaults/image.steps [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+Composite: Then an image with the src '$src' does not exist
+Then number of elements found by `By.xpath(.//img[@src='<src>'])` is = `0`
+
+Composite: Then an image with the src containing '$srcpart' exists
+Then number of elements found by `By.xpath(.//img[contains(@src,'<srcpart>')])` is = `1`
```
Comment (ikalinin1) [inline vividus-plugin-web-app/src/main/resources/steps/defaults/image.steps]: By.imageSrcPart(<srcpart>) here and every entry

File: vividus-plugin-web-app/src/main/resources/steps/defaults/image.steps [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+
+Composite: Then an image with the tooltip '$tooltip' and src containing '$srcpart' exists
+Priority: 1
+Then number of elements found by `By.xpath(.//img[contains(@src,'<srcpart>') and (@alt='<tooltip>' or @title='<tooltip>')])` is = `1`
+
+Composite: Then an image with the src '$imageSrc' and tooltip '$tooltip' exists
+Priority: 1
+Then number of elements found by `By.xpath(.//img[@src='<imageSrc>' and (@alt='<tooltip>' or @title='<tooltip>')])` is = `1`
```
Comment (ikalinin1) [inline vividus-plugin-web-app/src/main/resources/steps/defaults/image.steps]: why not update Tooltip ActionAttributeType to work with alt ?

**Pull request merged**